modules/javafx.web/src/main/java/com/sun/webkit/WebPage.java

```diff
@@ -618,12 +616,16 @@ public void setBackgroundColor(long frameID, int backgroundColor) {
         }
     }
 
+    public void setBackgroundColor(Color backgroundColor) {
+        log.fine("setBackgroundColor color: " + backgroundColor);
+        this.backgroundColor = backgroundColor;
+        setBackgroundColor(getColorInt32Value(backgroundColor));
+    }
+
     public void setBackgroundColor(int backgroundColor) {
-        this.backgroundColor = getColorFromHash(backgroundColor);
         lockPage();
         try {
-            log.fine("setBackgroundColor hash: " + backgroundColor +
-                    "(color: " + this.backgroundColor + ") " +
+            log.fine("setBackgroundColor int32: " + backgroundColor +
                    " for all frames");
```

Comment:
I don't know JavaFX's PlatformLogger but this should probably be:

```suggestion
            log.fine("setBackgroundColor int32: {} for all frames", backgroundColor);
```
or:

```suggestion
            if(log.isTraceEnabled()) {
                log.fine("setBackgroundColor int32: {} for all frames", backgroundColor);
            }
```

Even though this probably isn't something that's called very often :)

Reply:
Or just wrap the call in a test for `if (log.isLoggable(Level.FINE))`, which is similar to what was done earlier. In any case, this is just modifying the contents of the String and is otherwise preexisting code, so I'll leave it up to @jperedadnr as to whether he wants to change it.